organizers/2018-07-16.html

```diff
@@ -0,0 +1,134 @@
+---
+layout: organizers
+---
+<!-- Syntax reference: https://github.com/hakimel/reveal.js/blob/master/index.html -->
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>July 16, 2018</h2>
+    <img data-src="/img/django-logo-negative.png" style="width:40%" alt="Django logo" class="plain"></img>
+</section>
+
+<section data-background='#eee'>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Festicket</strong></li>
+        <li><strong>Thread</strong></li>
+        <li><strong>Time Out</strong></li>
+        <li><strong>Verve</strong></li>
+    </ul>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/festicket.svg" style="width:40%" alt="Festicket" class="plain"></img>
+    <p>
+        <em>Europe's largest music festival platform, and UK's 6th fastest growing private tech start up.<br>Transforming the way music fans discover, book tickets and travel packages for festivals around the world.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/thread.png" style="width:10%" alt="Thread" class="plain"></img>
+    <p>
+        <em>We help people to feel happier and more self-confident, by making it easy to dress well, using stylists and data science. Dedicated to maintaining a great working environment, minimising politics and maximising impact.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/timeout.png" style="width:40%" alt="Time Out" class="plain"></img>
+    <p>
+        <em>The trusted global platform that inspires and enables people to experience the best of the city. Operating in 107 cities, across 39 countries, it has an average monthly global audience reach of over 111 million across all platforms.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/verve.svg" style="width:40%" alt="Verve" class="plain"></img>
+    <p>
+        <em>We turn word of mouth into your best sales channel.</em>
```

Comment:
Again, just taken from the web site. Will send an email to see if there's a better logo/slogan they'd like us to use.